pandas/core/indexes/base.py

```diff
@@ -4036,6 +4036,10 @@ def contains(self, key):
         .. deprecated:: 0.25.0
             Use ``key in index`` instead of ``index.contains(key)``.
         """
+        warnings.warn(
+            "The 'contains' method is deprecated and will be removed in a "
+            "future versions. Use 'key in index' instead of "
+            "'index.contains(key)", FutureWarning, stacklevel=2)
```

Comment:
A couple small nitpicks:
- versions --> version
- mismatched single quote on the last line